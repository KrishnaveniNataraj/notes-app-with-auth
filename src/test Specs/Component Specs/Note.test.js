import Note from '../../components/Note';
import React from 'react';
import { shallow, mount } from 'enzyme';
import { createSerializer } from 'enzyme-to-json';

expect.addSnapshotSerializer(createSerializer({mode: 'deep'}));

let wrapper, note, handleRemoveNote;
beforeEach(() => {    
    note= {
        "noteTitle": "note1",
        "noteDescription": "note1 desc",
        "id": 1
        }
      handleRemoveNote = jest.fn();
    wrapper = shallow(
        <Note note={note} handleRemoveNote={handleRemoveNote} />
    );
});


test('Render Note component', () => {        
    expect(wrapper).toMatchSnapshot();
});