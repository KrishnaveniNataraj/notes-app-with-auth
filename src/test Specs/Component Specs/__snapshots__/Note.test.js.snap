// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Note component 1`] = `
<Note
  classes={
    Object {
      "deleteIcon": "Note-deleteIcon-1",
    }
  }
  handleRemoveNote={[MockFunction]}
  note={
    Object {
      "id": 1,
      "noteDescription": "note1 desc",
      "noteTitle": "note1",
    }
  }
/>
`;
